Podcast/ExternalProfileViewController.swift

```diff
@@ -280,9 +279,24 @@ class ExternalProfileViewController: ViewController, UITableViewDataSource, UITa
     func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         switch indexPath.section {
         case 0:
-            return RecommendedSeriesTableViewCell.recommendedSeriesTableViewCellHeight
+            guard let numSubs = numberOfRecommendedSeries() else { return RecommendedSeriesTableViewCell.recommendedSeriesTableViewCellHeight }
+            
+            if numSubs > 0 {
+                return RecommendedSeriesTableViewCell.recommendedSeriesTableViewCellHeight
+            }
+            else {
```

Comment:
you don't need these `else`'s here because you have returns in the if statements